### PR TITLE
test: add pause/stop/rename/ api tests

### PR DESCRIPTION
**Pull request by Letty5411**
Signed-off-by: letty.ll <letty.ll@alibaba-inc.com>

**1.Describe what this PR did**
add pause/stop/rename api tests

**2.Does this pull request fix one issue?** 

**3.Describe how you did it**

**4.Describe how to verify it**

**5.Special notes for reviews**

### CODE REVIEW

File: test/pouch_api_container_delete_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+func (suite *PouchAPIContainerDeleteSuite) SetUpTest(c *check.C) {
+	SkipIfFalse(c, environment.IsLinux)
+}
+
+// TestDeleteNonExisting tests deleting a non-existing container return error.
+func (suite *PouchAPIContainerDeleteSuite) TestDeleteNonExisting(c *check.C) {
+	cname := "TestDeleteNonExisting"
+	resp, err := request.Delete("/containers/" + cname)
+	c.Assert(err, check.IsNil)
+	c.Assert(resp.StatusCode, check.Equals, 204)
```
Comment (allencloud) [inline test/pouch_api_container_delete_test.go]: I think this should return a status code of 404 NOT FOUND.

File: test/pouch_api_container_create_test.go [hunk 2]
```diff
@@ -46,4 +47,8 @@ func (suite *PouchAPIContainerCreateSuite) TestCreateOk(c *check.C) {
 	c.Assert(err, check.IsNil)
 
 	c.Assert(got.ID, check.NotNil)
+
+	resp, err = request.Delete("/containers/" + cname)
```
Comment (allencloud) [inline test/pouch_api_container_create_test.go]: Why to delete this in `TestCreateOk `?

File: test/request/request.go [hunk 3]
```diff
@@ -98,6 +113,7 @@ func newRequest(method, url string, opts ...Option) (*http.Request, error) {
 			return nil, err
 		}
 	}
+	fmt.Println(req)
```
Comment (allencloud) [inline test/request/request.go]: remove this when ready.

File: test/pouch_api_container_create_test.go (hunk 2 above)
Comment (Letty5411) [reply test/pouch_api_container_create_test.go]: yes

File: test/pouch_api_container_rename_test.go [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	c.Assert(err, check.IsNil)
+	c.Assert(resp.StatusCode, check.Equals, 204)
+
+	resp, err = request.Delete("/containers/" + newname)
+	c.Assert(err, check.IsNil)
+	c.Assert(resp.StatusCode, check.Equals, 204)
+
+	resp, err = request.Delete("/containers/" + newname)
+	c.Assert(err, check.IsNil)
+	c.Assert(resp.StatusCode, check.Equals, 204)
```
Comment (sunyuan3) [inline test/pouch_api_container_rename_test.go]: Is it duplicate?

**Comment by allencloud** [comment]
LGTM

**Pull request merged**